1-intro/presentation.ipynb

```diff
@@ -72,6 +78,45 @@
     "# Go to www.menti.com and enter code: xxxx xxxx"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": "subslide"
+    },
+    "tags": []
+   },
+   "source": [
+    " - 3.7   Required for minimal version of ``astropy``\n",
+    " - 3.8   Walrus operator :=\n",
+    " - 3.9   Required for latest version of ``astropy``\n",
+    " - 3.10  More helpful error messages and pattern matching\n",
+    " - 3.11  Even better error messages\n",
+    " - 3.12  Better f-strings and better error messages again\n",
```

Comment:
If you want to have a list like this then it could at least mention the improvements regarding type annotations and Faster CPython.

Reply:
We don't mention or use type annotations in the materials, but the faster CPython is worth bringing up here. I will add it.